enterprise/frontend/src/embedding-sdk/components/private/SdkUsageProblem/SdkUsageProblemBanner.tsx

```diff
@@ -1,7 +1,8 @@
-import { useDisclosure } from "@mantine/hooks";
 import cx from "classnames";
 import { useState } from "react";
 
+import wrenchImage from "assets/img/sdk-banner-wrench.svg";
```

Comment:
This has colored shadows, so it's easier to use a SVG as an image than an icon.

<img src="https://github.com/user-attachments/assets/69ac6587-1b0a-43e9-b8c0-b1da7bdbbe74" width="100">